### PR TITLE
feat: warn user if no input provided & suggest --help

### PR DESCRIPTION
**Pull request by lili2311**
WIP

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -47,6 +47,10 @@ if (program.debug) {
   debugModule.enable(nameSpace);
 }
 
+if (!source) {
+  console.log('Detected no input. Please validate that the input provided to the CLI is an actual JSON. \nFor more options see `snyk-to-html --help`');
```
Comment (gitphill) [inline src/index.ts:51]: this prevents readInputFromStdin?

File: src/lib/snyk-to-html.ts [hunk 2]
```diff
@@ -63,10 +63,9 @@ class SnykToHtml {
                                remediation: boolean,
                                template: string,
                                summary: boolean): Promise<string> {
-    const promisedString = source ? readFile(source, 'utf8') : readInputFromStdin();
-    return promisedString
-      .then(promisedParseJSON)
-      .then(data => processData(data, remediation, template, summary));
+    const jsonData = await (source ? readFile(source, 'utf8') : readInputFromStdin());
```
Comment (gitphill) [inline src/lib/snyk-to-html.ts:66]: we want to console.log here if !!jsonData

Comment (lili2311) [reply src/lib/snyk-to-html.ts:66]: the next step does a nice message, do we want another check as well?

**Comment by aviadhahami** [comment]
Closing due to inactivity

**Pull request closed without merging**